appveyor.yml

```diff
@@ -20,28 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      VC_VERSION: 9
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      VC_VERSION: 9
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      VC_VERSION: 10
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      VC_VERSION: 10
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      VC_VERSION: 14
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      VC_VERSION: 14
-      CONDA_PY: 35
```

Comment:
Please see PR ( https://github.com/conda-forge/tk-feedstock/pull/6 ) for the fix.